contrib/agent/src/main/java/io/opencensus/contrib/agent/AgentMain.java

```diff
@@ -51,8 +53,15 @@ private AgentMain() {
    * @see java.lang.instrument
    */
   public static void premain(String agentArgs, Instrumentation inst) throws Exception {
```

Comment:
It must be "premain", cf. https://docs.oracle.com/javase/8/docs/api/java/lang/instrument/package-summary.html.